modules/ExperimentBrowser/src/client/ExperimentBrowser.coffee

```diff
@@ -626,6 +629,42 @@ class ExperimentBrowserController extends Backbone.View
 
 			window.open("/openExptInQueryTool?experiment=#{code}",'_blank')
 
+	handleGetLinkQueryToolClicked: => 
+			# Preparatory Logic to Get Code to Pass to Generate Link Route 
+			if @experimentController.model.get('lsLabels') not instanceof LabelList
```

Comment:
I see this was copied from "handleOpenInQueryToolClicked". Can we refactor to DRY this up?

Reply:
Of course!